modal/file_io.py

```diff
@@ -337,6 +345,49 @@ async def seek(self, offset: int, whence: int = 0) -> None:
         )
         await self._wait(resp.exec_id)
 
+    @classmethod
+    async def ls(cls, path: str, client: _Client, task_id: str) -> list[str]:
+        """List the contents of the provided directory."""
+        self = cls.__new__(cls)
```

Comment:
Is there any cleanup that needs to be done of these beyond simple garbage collect?

Reply:
don't think so, as these operations don't affect any state anywhere else